README.md

```diff
@@ -35,7 +35,11 @@ Se README i navkafka-docker-compose for mer info om hvordan man kjører den og k
 ## Lokal kjøring med Postgres
 For å kjøre mot lokal postgress så kan man kjøre DevLauncherPostgress.
 ```
-docker run --name postgres -p 5432:5432 -e POSTGRES_PASSWORD=postgres -e POSTGRES_USER=postgres -d familie-ba-mottak
+docker run --name familie-ba-mottak -p 5432:5432 -e POSTGRES_PASSWORD=test -d postgres
```

Comment:
Hvorfor ikke bare `-e POSTGRES_DB=familie-ba-mottak`?

Reply:
`docker run --name postgres --rm -p 5432:5432 -e POSTGRES_PASSWORD=test -e POSTGRES_USER=postgres -e POSTGRES_DB=familie-ba-mottak postgres` funker bra for meg

Reply:
Sant nok. Men det er jo ganske vennlig for nybegynnere å ha stegene listet ut - husker ihvertfall det hjalp meg da jeg skulle inn på ks-mottak første gang.